### PR TITLE
Sn 74 prescricao

### PR DESCRIPTION
**Pull request by andredavys**

### CODE REVIEW

**Comment by andredavys** [comment]
Realizado o CRUD. Falta tratar o relacionamento com nutricionista.

File: aplicacoes/sinutri/src/main/java/br/ufc/quixada/npi/sisat/controller/PacienteController.java [hunk 1]
```diff
@@ -493,4 +496,79 @@ public String cadastrarPaciente(@Valid PacienteExterno paciente, Model model,
...
+			@Valid Prescricao prescricao, BindingResult result, Model model, RedirectAttributes redirectAttributes){
+		
+		if(!result.hasErrors())
+			pacienteService.editarPrescricao(prescricao);
+		
+		return "redirect:/paciente/"+idPaciente+"/Prescricao/"+prescricao.getId()+"/";
+	}
+	
+	@RequestMapping(value="/{idPaciente}/Prescricao/{idPrescricao}/", method = RequestMethod.GET)
+	public String visualizarPrescricao(@PathVariable("idPaciente") Long idPaciente, @PathVariable("idPrescricao") Long idPrescricao, Prescricao prescricao,
```
Comment (brunomateus) [inline aplicacoes/sinutri/src/main/java/br/ufc/quixada/npi/sisat/controller/PacienteController.java]: Codacy Issue found: [Avoid reassigning parameters such as 'prescricao'](https://www.codacy.com/app/NPI-UFC/sinutri/file/2656234348/issues/source?bid=3258170&fileBranchId=3327579#l565)

File: aplicacoes/sinutri/src/main/java/br/ufc/quixada/npi/sisat/controller/PacienteController.java [hunk 2]
```diff
@@ -10,6 +10,7 @@
 import javax.servlet.http.HttpSession;
 import javax.validation.Valid;
 
+import org.bouncycastle.jce.provider.JDKAlgorithmParameters.IDEAAlgorithmParameters;
```
Comment (brunomateus) [inline aplicacoes/sinutri/src/main/java/br/ufc/quixada/npi/sisat/controller/PacienteController.java]: Codacy Issue found: [Avoid unused imports such as 'org.bouncycastle.jce.provider.JDKAlgorithmParameters.IDEAAlgorithmParameters'](https://www.codacy.com/app/NPI-UFC/sinutri/file/2656234348/issues/source?bid=3258170&fileBranchId=3327579#l13)

File: aplicacoes/sinutri/src/main/java/br/ufc/quixada/npi/sisat/controller/PacienteController.java [hunk 3]
```diff
@@ -34,6 +35,8 @@
 import br.ufc.quixada.npi.sisat.model.Paciente;
 import br.ufc.quixada.npi.sisat.model.PacienteExterno;
 import br.ufc.quixada.npi.sisat.model.Pessoa;
+import br.ufc.quixada.npi.sisat.model.Prescricao;
+import br.ufc.quixada.npi.sisat.model.Servidor;
```
Comment (brunomateus) [inline aplicacoes/sinutri/src/main/java/br/ufc/quixada/npi/sisat/controller/PacienteController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.sisat.model.Servidor'](https://www.codacy.com/app/NPI-UFC/sinutri/file/2656234348/issues/source?bid=3258170&fileBranchId=3327579#l39)

**Pull request closed without merging**